pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -907,19 +907,19 @@ cdef class _Timedelta(ABCTimedelta):
         Examples
         --------
         >>> td = pd.Timedelta('1 days 2 min 3 us 42 ns')
-        >>> td.resolution
+        >>> td.resolution_string
```

Comment:
I *think* this is a public method, no?

Reply:
this is unrelated to the rest of the PR, just incorrect ATM.  we changed the method name a while back and apparently didnt update the docstring

BTW i commented on one of your PRs to ask: where did the one-letter abbreviations currently used in this function come from?  id like a name for them to distinguish them from the 2-3 letter abbrevs used elsewhere

Reply:
not idea where the 1 letter names came from